src/coreclr/jit/lsraxarch.cpp

```diff
@@ -1324,6 +1324,23 @@ int LinearScan::BuildCall(GenTreeCall* call)
             ctrlExprCandidates = availableIntRegs & ~(RBM_ARG_REGS);
         }
         srcCount += BuildOperandUses(ctrlExpr, ctrlExprCandidates);
+        if (call->gtCallType == CT_INDIRECT)
+        {
```

Comment:
Should you add the same comment here that you added in lsraarmarch.cpp?

Reply:
This should be unnecessary if the arg is made a late arg (expected for all register arguments).